MarsMission/Controllers/LandingViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  LandingViewController.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/03.
+//
+
+import UIKit
+
+class LandingViewController: UIViewController {
+	
+	@IBOutlet weak var dailyWeatherReport: UIButton!
+	@IBOutlet weak var threeDayWeatherReport: UIButton!
+	@IBOutlet weak var seasonalWeatherReport: UIButton!
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		navigationItem.title = keys.navigationTitle.landingtitle
+		dailyWeatherReport.layer.cornerRadius = 30
+		threeDayWeatherReport.layer.cornerRadius = 30
+		seasonalWeatherReport.layer.cornerRadius = 30
+		
+		ServiceImplementation.shared.getWeatherData() { result in
+			switch result {
+				case .success(let results):
```

Comment:
Please fix spacing and indentation

Reply:
<img width="1035" alt="Screenshot 2020-11-06 at 10 50 24" src="https://user-images.githubusercontent.com/55865505/98346015-19451e80-201e-11eb-948b-f2c8c36a2b78.png">